### PR TITLE
feat(webpack): Improve performance of webpack build

### PR DESCRIPTION
**Pull request by christopherthielen**
- use ts-loader + happypack
    - update to webpack 3.4.x
    - use babel-plugin-angularjs-annotate instead of ng-annotate

@jrsquared @anotherchrisberry @icfantv @danielpeach

### CODE REVIEW

**Comment by christopherthielen** [comment]
I need to update the other webpack configs for `core`, `amazon`, etc

File: webpack.common.js [hunk 1]
```diff
@@ -1,16 +1,33 @@
...
+    "BAKERY_DETAIL_URL",
+    "AUTH_ENDPOINT",
+    "AUTH_ENABLED",
+    "NETFLIX_MODE",
+    "CHAOS_ENABLED",
+    "FIAT_ENABLED",
+    "ENTITY_TAGS_ENABLED",
+    "DEBUG_ENABLED",
+    "CANARY_ENABLED",
+    "INF_SEARCH_ENABLED",
```
Comment (anotherchrisberry) [inline webpack.common.js]: single quotes

**Comment by anotherchrisberry** [comment]
hurry up already

**Comment by jrsquared** [comment]
!!!

**Comment by icfantv** [comment]
Omg omg omg omg omg omg

**Pull request merged**